src/codegeneration/ParenTrait.js

```diff
@@ -129,5 +130,16 @@ export function ParenTrait(ParseTreeTransformerClass) {
       }
       return new ArrayLiteralExpression(tree.location, elements);
     }
+
+    transformPropertyNameAssignment(tree) {
+      let name = this.transformAny(tree.name);
+      let value = this.transformAny(tree.value);
+      if (value.type === COMMA_EXPRESSION) {
```

Comment:
So how does this work? The test case has `AssignmentExpression : ArrowFunction` and the code tests COMMA_EXPRESSION.